vault/programs/vault/src/context.rs

```diff
@@ -0,0 +1,323 @@
+use crate::zeta_context::*;
+use crate::*;
+
+#[derive(Accounts)]
+#[instruction(vault_name: String, vault_lamports: u64, bumps: VaultBumps)]
+pub struct InitializeVault<'info> {
+    // vault Authority accounts
+    #[account(mut)]
+    pub vault_authority: Signer<'info>,
+    // vault Accounts
+    #[account(init,
+        seeds = [vault_name.as_bytes()],
+        bump = bumps.vault,
+        payer = vault_authority)]
+    pub vault: Box<Account<'info, Vault>>,
+    // This is the PDA that holds SOL to pay for the margin account
+    #[account(mut,
+        seeds = [vault_name.as_bytes(), b"payer"],
+        bump = bumps.vault_payer)]
+    pub vault_payer: AccountInfo<'info>,
+    // TODO Confirm USDC mint address on mainnet or leave open as an option for other stables
+    #[account(constraint = usdc_mint.decimals == PLATFORM_PRECISION as u8)]
```

Comment:
Can use build rust flags for the correct usdc address you expect.

Reply:
https://github.com/zetamarkets/zeta-options/blob/main/zeta/programs/zeta/src/address.rs#L1-L7